README.md

```diff
@@ -20,6 +20,17 @@ https://decred.org
 
 https://decred.org/downloads
 
+Core software:
+
+* dcrd: a Decred full node daemon (this)
+* [dcrwallet](https://github.com/decred/dcrwallet): a CLI Decred wallet daemon
+* [politeia](https://github.com/decred/politeia): a daemon for the Decred proposal system
+
+Bundles:
+
+* [Decrediton](https://github.com/decred/decrediton): a GUI bundle for `dcrd`, `dcrwallet`, and `politeia`
```

Comment:
This isn't accurate.  The politeia daemon is not something the user runs. It's a public remote service that Decrediton interacts with.

Reply:
Removed `politeia`